x-pack/legacy/plugins/epm/public/components/package_card.tsx

```diff
@@ -26,31 +26,16 @@ export function PackageCard({
   name,
   title,
   version,
-  icon: iconUrl,
```

Comment:
Old property name (now `icons`) and we weren't using it. Will introduce `icons` later